image_stacking/mod_mergeimages_160929.py

```diff
@@ -1,48 +1,50 @@
-#https://bespokeblog.wordpress.com/category/programming/python/
+# https://bespokeblog.wordpress.com/category/programming/python/
 import sys
 from PIL import Image
 import glob
 
 # REPLACE "im1.jpg" with imgList[0]
 # REPLACE imdir+'im%d.jpg'%(imnum) with imgList values
-'''
```

Comment:
If we are using double quotes for everything, block comments should be single quotes.

Reply:
also, we can toss old/commented-out code